cpp/autosar/test/rules/A13-3-1/test.cpp

```diff
@@ -61,9 +62,17 @@ A b(a);
 void F1(int &) = delete; // COMPLIANT by exception
 
 struct B {
-  template <typename T,
-            std::enable_if_t<!std::is_same<T, B>::value> * = nullptr>
-  B(T &&value) {}
+  template <
+      typename T,
+      std::enable_if_t<!std::is_same<
+          std::remove_cv_t<std::remove_reference_t<T>>, A>::value> * = nullptr>
+  B(T &&value) {} // COMPLIANT[FALSE_POSITIVE] - by exception
 };
 
-int main() {}
\ No newline at end of file
+int main() {}
+
+class C {
+public:
+  C() {} // COMPLIANT by exception
+  template <typename T> C(T &&) {}
```

Comment:
```suggestion
  C() {} 
  template <typename T> C(T &&) {} // COMPLIANT by exception
```

The alert is now on the move constructor so we need to adjust the COMPLIANT annotation.